src/uiTest/resources/project.rhino.js

```diff
@@ -402,6 +402,18 @@ function Project(underlyingProject) {
     invokeActionAndWait('GitMachete.CheckoutSelectedAction', ACTION_PLACE_CONTEXT_MENU, { SELECTED_BRANCH_NAME: branchName });
   };
 
+  this.checkoutNextBranch = function() {
+    invokeActionAndWait('GitMachete.NextBranchCheckoutAction', "",{});
```

Comment:
Nit
```suggestion
    invokeActionAndWait('GitMachete.NextBranchCheckoutAction', /* actionPlace */ "", {});
```
Here and in the others